README.md

```diff
@@ -154,15 +154,27 @@ piece of state must be named `routing`** (unless you provide a custom
 
 ### `UPDATE_PATH`
 
-An action type that you can listen for in your reducers.
+An action type that you can listen for in your reducers to be notified of
+route updates.
 
-### `updatePath(path, noRouterUpdate)`
+### `pushPath(path, state = {}, avoidRouterUpdate = false)`
 
-An action creator that you can use to update the current URL. Just
-pass it a string like `/foo/bar?param=5`.
+An action creator that you can use to update the current URL and update the
+browser history. Just pass it a string like `/foo/bar?param=5` as the
+`path` argument.
+
+You can optionally pass a state to this action creator to update the state of the
+current route.
 
 The `noRouterUpdate`, if `true`, will stop react-router from reacting
 to this and all future URL changes. Pass `false` to make it start
```

Comment:
We should remove "and all future" and the next sentence ("Pass `false` to make it start..."). `avoidRouterUpdate` is only for the specific action, not for later actions.